pkg/lib/cockpit-components-table.tsx

```diff
@@ -24,6 +24,8 @@ import {
     ExpandableRowContent,
     Table, Thead, Tbody, Tr, Th, Td,
     SortByDirection,
+    TdProps, ThProps, TrProps, TableProps,
+    OnSelect,
```

Comment:
Why? I don't like making this distrinction, tbh. My impression is that "import type" only exists to work around problems with certain module or build systems.

Reply:
We don't need them at runtime, so they shouldn't be in the generated bundle. Not sure if it makes a difference here, but we did have cases of either circular imports, or adding dead weight to a bundle because we *only* imported types from a file. That isn't the case here, so it's mostly just about getting used to `import type`.

Reply:
Done!